charts/kyverno/templates/reports-controller/flowschema.yaml

```diff
@@ -0,0 +1,92 @@
+{{- if .Values.reportsController.apiPriorityAndFairness }}
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta3
+kind: FlowSchema
+metadata:
+  name: {{ template "kyverno.reports-controller.flowSchema" . }}
+  labels:
+    {{- include "kyverno.reports-controller.labels" . | nindent 4 }}
+spec:
+  priorityLevelConfiguration:
+    name: {{ template "kyverno.reports-controller.priorityLevelConfiguration" . }}
+  rules:
```

Comment:
I wonder how configurable we want this to be

Reply:
I added the rules defined in `Role` and `ClusterRole` for the reports service account.